compiler/ballerina-treegen/src/main/resources/syntax_tree_descriptor.json

```diff
@@ -3053,7 +3053,8 @@
                 },
                 {
                     "name": "waitFutureExpr",
-                    "type": "Node"
+                    "type": "ExpressionNode",
+                    "occurrences": "MULTIPLE_SEPARATED"
```

Comment:
We can't make it a list. Wait action can be one of: 
```
wait-action :=
   single-wait-action
   | multiple-wait-action
   | alternate-wait-action
```

In all cases, wait-keyword is common, but the future-expression changes:
```
wait-action := wait-keyword ( expression |  multiple-waits | alternate-waits)
```

Reply:
fixed with 9bebd1a.